### PR TITLE
Updated to stable rc2 bits

### PR DESCRIPTION
**Pull request by thecodejunkie**
### Prerequisites

- [X] I have written a descriptive pull-request title
- [X] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
- [X] I have verified that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
- [X] I have provided test coverage for my change (where applicable)

### Description
Updated to the stable rc2 bits now that they are out

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-    "projects": [ "src", "test", "samples" ]
+    "projects": [ "src", "test", "samples" ],
+    "sdk": {
+        "version": "1.0.0-preview1-002702"
```
Comment (khellang) [inline global.json]: Is this 'stable'? I have `1.0.0-rc2-002392` :confused:

Comment (thecodejunkie) [inline global.json]: That corresponds to the version that you get of the tooling etc when you install the bits from https://www.microsoft.com/net/core

> dotnet --info
> .NET Command Line Tools (1.0.0-preview1-002702)

Comment (jchannon) [inline global.json]: Yup this is the RC2 stable version that MS recommend to put in global.json

Comment (khellang) [inline global.json]: > That corresponds to the version that you get of the tooling etc when you install the bits from https://www.microsoft.com/net/core

Huh. I literally just installed that 2 minutes ago and get

> $ dotnet --version
> 1.0.0-rc2-002392

Think I might have some old :hankey: lying around. The uninstall probably didn't work.

Comment (khellang) [inline global.json]: > Yup this is the RC2 stable version that MS recommend to put in global.json

@jchannon Where is this recommendation?

Comment (jchannon) [inline global.json]: @khellang here https://blogs.msdn.microsoft.com/visualstudio/2016/05/16/announcing-updated-web-development-tools-for-asp-net-core-rc2/

**Comment by jchannon** [comment]
LGTM :shipit:

**Comment by phillip-haydon** [comment]
WOW, worlds most boring PR :D can't see anything wrong or inconsistent in the changes. 

I'll let something else merge this but I see no issues.

**Pull request merged**